ReactCommon/DEFS

```diff
@@ -2,8 +2,7 @@
 
 # Building is not supported in OSS right now
 def rn_xplat_cxx_library(name, **kwargs):
-  cxx_library(name = name)
-
+  cxx_library(name, **kwargs)
```

Comment:
What is this change for?

Reply:
I'm still playing around. The android tests are failing randomly.
This fixed the fetch for me on the master.

Reply:
As I don't know all the steps needed to run tests I'm just pushing the branch and see the results.
Then fixing the related part of the script locally.